server/src/actors/events.rs

```diff
@@ -94,13 +79,16 @@ enum ProcessingError {
     StoreFailed(#[cause] StoreError),
 
     #[fail(display = "sending failed due to rate limit ({}s)", _0)]
-    RateLimited(u64),
+    RateLimited(u64, String),
 
     #[fail(display = "event exceeded its configured lifetime")]
     Timeout,
 
     #[fail(display = "shutdown timer expired")]
     Shutdown,
+
+    #[fail(display = "upstream rate limited")]
+    UpstreamRateLimited,
```

Comment:
Please keep this merged with `RateLimited(u64, String)`.

Reply:
ok, done